zebrad/src/commands.rs

```diff
@@ -50,28 +50,9 @@ pub enum ZebradCmd {
 }
 
 impl ZebradCmd {
-    /// Returns true if this command sends output to stdout.
-    ///
-    /// For example, `Generate` sends a default config file to stdout.
-    ///
-    /// Usage note: `abscissa_core::EntryPoint` stores an `Option<ZerbradCmd>`.
-    /// If the command is `None`, then abscissa writes zebrad usage information
-    /// to stdout.
-    pub(crate) fn uses_stdout(&self) -> bool {
-        match self {
-            // List all the commands, so new commands have to make a choice here
-            Generate(_) | Help(_) | Revhex(_) | Version(_) => true,
-            Seed(_) | Start(_) => false,
-        }
-    }
-
     /// Returns true if this command is a server command.
     ///
     /// For example, `Start` acts as a Zcash node.
-    ///
-    /// Usage note: `abscissa_core::EntryPoint` stores an `Option<ZerbradCmd>`.
-    /// If the command is `None`, then abscissa prints zebrad's usage
-    /// information, then exits.
```

Comment:
I removed this because it's not strictly speaking a usage note on `ZebradCmd` but about abscissa, and the new tracing initialization logic doesn't have to handle the `None` case differently.